packages/generators/types/index.ts

```diff
@@ -0,0 +1,37 @@
+export interface ISchemaProperties {
+	additionalProperties?: boolean;
+	definitions?: object;
+	properties?: object;
+	type?: string;
+}
+
+export interface IWebpackOptions {
```

Comment:
These types need to be written as they will be, all of them are strings in the ast types, but here they are specified. Check webpack validation schema for definition of types

Reply:
You mean everything from this [file](https://github.com/webpack/webpack/blob/master/schemas/WebpackOptions.json)?

Reply:
Yes that's right

Reply:
Base your values on that file, avoid using `any`